alpha/lib/model/MediaServer.php

```diff
@@ -190,4 +190,8 @@ public function setIsExternalMediaServer($v)
 		$this->isExternalMediaServer = $v;
 	}
 	
+	public function getIsExternalMediaServer()
```

Comment:
U R right ... I used to use it but I changed the code and it is no longer needed. I will remove this function.